app/src/main/java/com/github/weg_li_android/data/model/District.kt

```diff
@@ -0,0 +1,9 @@
+package com.github.weg_li_android.data.model
+
+data class District(
+    val id: Int,
+    val zip: String,
+    val name: String,
+    val email: String,
+    val prefix: String
```

Comment:
Instead of prefix i would add the `aliases` array, otherwise it's basically not possible to use the app in Munich.

Reply:
What is aliases? Looking at the json most at the beginning are either [] or null

Reply:
I just found an entry: `pp-mue.muenchen.pi11@polizei.bayern.de","pp-mue.muenchen.pi12@polizei.bayern.de`

Looks to me like the alternative email addresses? I'll add that.

If we have no use for the prefix, I can remove it. I only picked data, that seemed interesting.

Reply:
Munich does not have a central e-mail address to handle charges. People have to know which police station they have to e-mail it to. At least, we can give a little guidance based on ZIPs.